Makefile

```
@@ -36,13 +42,17 @@ github.repos: \
 	curl -sS https://api.github.com/repos/supabase/$* \
-	> $(REPO_DIR)/web/src/data/repos/$*.json
+	> $(WWW_DIR)/data/repos/$*.json
 
+.PHONY: github.traction
 github.traction:
-	cd "$(REPO_DIR)"/web && \
+	cd "$(WWW_DIR)" && \
 	npm run traction
 
+.PHONY: dev
 dev:
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)